README.md

```diff
@@ -1,2 +1,71 @@
-# movesta-avm-widget
-NOTE: This is work in progress, not ready for use. Widget for embedding Movesta's AVM on any website.
+# 🏠 Movesta AVM Widget
+
+A third-party integration of Movesta's automated valuation model for real estate in Sweden. 
+
+Estimate home values as of today using an easy-to-integrate website widget, a great tool for customer acquisition. Shows an estimate and range of the value of apartments (lägenhet), detached (villor), townhouses (radhus), and holiday homes (fritidshus) across Sweden via basic information like location, living area, and number of rooms.
+
+## 📦 Installation
+
+Include the widget via a script tag. For usage with React, follow this guide: [4 Ways of Adding External JS Files in ReactJS](https://betterprogramming.pub/4-ways-of-adding-external-js-files-in-reactjs-823f85de3668). A proper npm package will be available in the future.
+
+```html
+<!-- <script 
+  src="https://cdn.jsdelivr.net/gh/MovestaDev/movesta-avm-widget@development/static/movesta-avm-widget.js"
+  type="text/javascript">
+</script> -->
+
+<script type="text/javascript">
+  const myStyle = {
+    fontFamily: 'inherit',
+    accentColor: '#03396c',
+    accentDark: '#03396c',
+  };
+
+  MovestaAVMWidget.init({
```

Comment:
`termsAndConditions` and `onComplete` must also be specified, right? Best to include them in the example :)